Smarty/templates/QuickCreate.tpl

```diff
@@ -33,10 +33,7 @@
 				</tr>
 			{/foreach}
 			{if $MODULE eq 'Products' || $MODULE eq 'Services'}
-				{if $rowiteration==2}
-					<tr><td colspan="2" class="dvtCellInfo">&nbsp;</td></tr>
-				{/if}
-				{assign var=rloopit value=$rowiteration}
+				{assign var=rloopit value=2}
```

Comment:
I don't think this will work as there is no way of you knowing that it will balance correctly the row. Whether you need to close the TR or not depends on the number of fields before you get to the taxes section and that is determined by the fields activated by the admin user, so you cannot hardcode it to 2, you must know what iteration you are on.